roop/core.py

```diff
@@ -129,6 +143,26 @@ def preview_video(video_path):
         img_label.image = photo_img
         img_label.pack()
 
+        # Preview
+        preview.update(frame)
+        amount_of_frames = cap.get(cv2.CAP_PROP_FRAME_COUNT)
+        
+        def update_slider(frame_number):
+            preview.update(get_video_frame(video_path, frame_number))
+
+        preview.init_slider(amount_of_frames, update_slider)
+
+        def test_handler():
```

Comment:
Please no nested def... move it out - can you rename it preview_handler() aswell?